app/src/main/java/com/gh4a/activities/IssueListActivity.java

```diff
@@ -94,6 +94,7 @@ public static Intent makeIntent(Context context, String repoOwner, String repoNa
     private int mSelectedParticipatingStatus = 0;
 
     private FloatingActionButton mCreateFab;
+    private MenuItem removeFilterButton;
```

Comment:
`mRemoveFilterButton`

Reply:
Done.
I like this naming convention so much that I keep forgetting about it :joy:

Reply:
Nowadays, when writing Kotlin code, I omit the `m` prefix as well, but let's keep the code base consistent ;-)

Reply:
I agree I agree, it's just something very unnatural if you aren't used to it😅
Since you mentioned Kotlin, what about introducing it in the app? In addition to helping reduce boilerplate, it would allow us to use coroutines which are IMO way easier to approach than RxJava.

Reply:
> Since you mentioned Kotlin, what about introducing it in the app? In addition to helping reduce boilerplate, it would allow us to use coroutines which are IMO way easier to approach than RxJava.

As much as I love Kotlin, that would be a huge rewrite (since one would need to rewrite the bindings for that as well). I'm happy to make you the maintainer of both, but I'm afraid this is too much effort for my personal time budget. 😔

Reply:
I didn't mean to rewrite the whole app using Kotlin + coroutines, that would be really too much 😅
My idea is to at least make Kotlin available for developing new parts and for gradually migrating existing portions of the codebase to Kotlin as we work on them.
Regarding coroutines, we wouldn't need to rewrite the bindings to use them since the Kotlin coroutines library [offers some utilities for RxJava 2 interoperability](https://kotlinlang.org/api/kotlinx.coroutines/kotlinx-coroutines-rx2/). I'm not sure on what could be the best way to integrate the two libraries (and if it's a good thing to do), but having Kotlin in the project would give us the opportunity to experiment.